templates/docs/examples/layouts/application/JAAS.html

```diff
@@ -45,6 +45,14 @@
   .is-jaas-background {
     background: #003b4e !important;
   }
+
+  .is-jaas-background .p-side-navigation__link[aria-current="page"] {
```

Comment:
This is such an outdated example (in terms of colours), maybe it would be better to get rid of custom background and make it plain dark?

Reply:
Removed the JAAS custom background, it is now plain dark